src/Title.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
```

Comment:
Not sure you need a custom component for the title but still you have to clean the code.